buffer.go

```diff
@@ -30,11 +27,17 @@ func (b *memBuffer) Less(i, j int) bool { return b.less(b.ents[i].Key(), b.ents[
 func (b *memBuffer) Swap(i, j int)      { b.ents[i], b.ents[j] = b.ents[j], b.ents[i] }
 
 func (b *memBuffer) Reset() {
+	for _, e := range b.ents {
+		e.Release()
+	}
 	b.size = 0
 	b.ents = b.ents[:0]
 }
 
 func (b *memBuffer) Free() {
+	for _, e := range b.ents {
```

Comment:
minor then, can you just change this to:

```go
b.Reset()
b.ents = nil
```

will merge after that!

Reply:
Changed 👍